ril_subscription.conf

```diff
@@ -371,3 +371,10 @@ socket=/dev/socket/rild
 # Default 30000
 #
 #cellInfoIntervalLongMs=30000
+
+# Configure the ril request originated in ril_voicecall_set_udub. Some
+# devices like the Volla phone halts the modem when asking if the user
+# is busy through this function. This produces that, if you cancel an
+# awaiting call because of an on going one, both calls will be hang up.
+#
+#rilRequestOnSetUdub=13
```

Comment:
AFAICT the default for RIL_REQUEST_UDUB  is 17, please mention that in a way similar to how it's done for other entries.